docs/releases.rst

```diff
@@ -34,6 +34,23 @@ support for :ref:`cpu.vendor-name</spec/hardware/cpu>` and
 The linting of tests, plans and stories has been extended by detecting
 duplicate ids.
 
+The ``tmt-report-result`` command has been updated to generate a
+``results.yaml`` file with content which differ from the
+previously generated ``restraint-result`` file.
+
+The behaviour for existing tests which already utilise the
+``tmt-report-result`` command shall remain unchanged, in that the
+overall result returned by tmt shall be determined by parsing the
+``results.yaml`` file and selecting the result property with the
+value which resides highest on the hierarchy of ``skip``, ``pass``,
+``warn``, ``fail``.
+
+By setting ``result: custom`` in the ``fmf`` file the results.yaml
```

Comment:
I would make it more concrete, we use `fmf` files for plans, tests and stories, so in `tmt test metadata`